### PR TITLE
Code refactor for queries

### PR DESCRIPTION
**Pull request by ajaypratap003**
### Type of change

- Refactoring

### Description
1. Refactor garphql queries into separate file
2. Refactor query function and create separate function for filter, and sorting
3. Added code coverage configuration 
4. Added new yarn/npm command to run unit test with code coverage i.e. test:coverage
5. Implemented absolute path for file system
6. Create constant file and moved hard code values into constant file
7. Changed folder name as camel case consistency

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/queries/Address.ts [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+  }
+`;
+
+const PURGE_ADDRESS = gql`
+  mutation purge_addr($a: ObjectMeta_v1_Input!) {
+    purgeAddress(input: $a)
+  }
+`;
+
+const ALL_ADDRESS_FOR_ADDRESS_SPACE_FILTER = (
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Address.ts]: I think we can modify this name as FILTER_FOR_ADDRESSES.

File: console/console-init/ui/src/queries/Address.ts [hunk 2]
```diff
@@ -0,0 +1,696 @@
...
+      status = "Pending";
+    } else {
+      status = statusValue;
+    }
+    filter += "`$.Status.Phase` = '" + status + "'";
+  }
+  return filter;
+};
+
+const ALL_ADDRESS_FOR_ADDRESS_SPACE_SORT=(sortBy?: ISortBy)=>{
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Address.ts]: We can rename this as SORT_STRING_FOR_ADDRESSES.

File: console/console-init/ui/src/queries/Address.ts [hunk 3]
```diff
@@ -0,0 +1,696 @@
...
+  page: number,
+  perPage: number,
+  name?: string,
+  namespace?: string,
+  filterNames?: any[],
+  typeValue?: string | null,
+  statusValue?: string | null,
+  sortBy?: ISortBy
+) => {
+  let filter =ALL_ADDRESS_FOR_ADDRESS_SPACE_FILTER(name,namespace,filterNames,typeValue,statusValue);
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Address.ts]: It seems linting is not working.

File: console/console-init/ui/src/queries/Address.ts [hunk 4]
```diff
@@ -0,0 +1,696 @@
...
+            Units
+          }
+        }
+      }
+    }
+  `;
+  return ALL_ADDRESS_FOR_ADDRESS_SPACE;
+};
+
+const CURRENT_ADDRESS_SPACE_PLAN = (name?: string, namespace?: string) => {
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Address.ts]: This should be part of addressSpaces.queries

File: console/console-init/ui/src/queries/Address.ts [hunk 5]
```diff
@@ -0,0 +1,696 @@
...
+            Units
+          }
+        }
+      }
+    }
+    `;
+  return ADDRESSDETAIL;
+};
+
+const ADDRESS_LINKS_FILTER=(
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Address.ts]: We can rename this as FILTER_STRING_FOR_ADDRESS_LINKS.

File: console/console-init/ui/src/queries/Address.ts [hunk 6]
```diff
@@ -0,0 +1,696 @@
...
+
+  if (filterRole && filterRole.trim() != "") {
+    filterForLink +=
+      "`$.Spec.Role` = '" + filterRole.trim().toLowerCase() + "' ";
+  }
+
+  return {filter,filterForLink};
+}
+
+const ADDRESS_LINKS_SORT=(sortBy?: ISortBy,)=>{
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Address.ts]: We can rename this as SORT_STRING_FOR_ADDRESS_LINKS

File: console/console-init/ui/src/queries/Connection.ts [hunk 7]
```diff
@@ -0,0 +1,542 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import gql from "graphql-tag";
+import { ISortBy } from "@patternfly/react-table";
+
+const ALL_CONECTION_LIST_FILTER=(
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Connection.ts]: We should rename this as FILTER_STRING_FOR_CONNECTIONS

File: console/console-init/ui/src/queries/Connection.ts [hunk 8]
```diff
@@ -0,0 +1,542 @@
...
+          "(`$.Spec.ContainerId` = '" + containerValue + "')";
+      else
+        filter +=
+          "(`$.Spec.ContainerId` LIKE '" + containerValue + "%')";
+    }
+  }
+  return filter;
+};
+
+const ALL_CONECTION_LIST_SORT=(sortBy?: ISortBy)=>{
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Connection.ts]: We should rename this as SORT_STRING_FOR_CONNECTIONS

File: console/console-init/ui/src/queries/Connection.ts [hunk 9]
```diff
@@ -0,0 +1,542 @@
...
+        break;
+      default:
+        break;
+    }
+    orderBy += sortBy.direction;
+  }
+  return orderBy;
+};
+
+const RETURN_ALL_CONECTION_LIST = (
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Connection.ts]: There is a typo in all places of CONNECTION

File: console/console-init/ui/src/queries/Connection.ts [hunk 10]
```diff
@@ -0,0 +1,542 @@
...
+  page: number,
+  perPage: number,
+  hostnames: any[],
+  containers: any[],
+  name?: string,
+  namespace?: string,
+  sortBy?: ISortBy
+) => {
+  let filter =ALL_CONECTION_LIST_FILTER(hostnames,containers,name,namespace);
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Connection.ts]: It seems linting is not working for .ts extension files.

File: console/console-init/ui/src/queries/Connection.ts [hunk 11]
```diff
@@ -0,0 +1,542 @@
...
+        break;
+    }
+    if (sortBy.direction && orderBy !== "") {
+      orderBy += sortBy.direction;
+    }
+  }
+  return orderBy;
+};
+
+const CONNECTION_LINKS_FILTER=(
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Connection.ts]: We can rename this as FILTER_STRING_FOR_CONNECTION_LINKS

File: console/console-init/ui/src/queries/Connection.ts [hunk 12]
```diff
@@ -0,0 +1,542 @@
...
+            Units
+          }
+        }
+      }
+    }
+    `;
+  return CONNECTION_DETAIL;
+};
+
+const CONNECTION_LINKS_SORT=( sortBy?: ISortBy,)=>{
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Connection.ts]: We can rename this as SORT_STRING_FOR_CONNECTION_LINKS

File: console/console-init/ui/src/queries/Connection.ts [hunk 13]
```diff
@@ -0,0 +1,542 @@
...
+  perPage: number,
+  filterNames: any[],
+  filterAddresses: any[],
+  addressSpaceName?: string,
+  addressSpaceNameSpcae?: string,
+  connectionName?: string,
+  sortBy?: ISortBy,
+  filterRole?: string
+) => {
+  const {filter,filterForLink}=CONNECTION_LINKS_FILTER(filterNames,filterAddresses,addressSpaceName,addressSpaceNameSpcae,connectionName,filterRole);
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/queries/Connection.ts]: It seems linting is not working for .ts extension files.

File: console/console-init/ui/src/queries/Address.ts (hunk 1 above)
Comment (ajaypratap003) [reply console/console-init/ui/src/queries/Address.ts]: done

**Comment by ajaypratap003** [comment]
changes have implemented, please check.

**Pull request merged**